### PR TITLE
Introduce WalletCreator helper

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**
This PR contains #839

### CODE REVIEW

**Comment by marekkirejczyk** [comment]
Needs rebase

File: universal-login-relayer/test/helpers/WalletCreator.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  constructor(relayerUrl: string, private wallet: Wallet) {
+    this.http = http(fetch)(relayerUrl);
+  }
+
+  async getRelayerConfig(): Promise<PublicRelayerConfig> {
+    this.relayerConfig = this.relayerConfig || (await this.http('GET', '/config')).config;
+    return this.relayerConfig!;
+  }
+
+  private async getEnsService() {
```
Comment (marekkirejczyk) [inline universal-login-relayer/test/helpers/WalletCreator.ts]: `fetchEnsService`? If it is asynchronous and takes complex interactions (Http, blockchain) it should be `fetch...`. If it is a local getter it should be `get...`.

File: universal-login-relayer/test/helpers/WalletCreator.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import fetch from 'node-fetch';
+import {Wallet, Contract, utils} from 'ethers';
+import {http, HttpFunction, PublicRelayerConfig, createKeyPair, calculateInitializeSignature, computeContractAddress, TEST_GAS_PRICE} from '@universal-login/commons';
+import {encodeInitializeWithRefundData} from '@universal-login/contracts';
+import ProxyCounterfactualFactory from '@universal-login/contracts/build/ProxyCounterfactualFactory.json';
+import ENSService from '../../lib/integration/ethereum/ensService';
+
+export class WalletCreator {
```
Comment (marekkirejczyk) [inline universal-login-relayer/test/helpers/WalletCreator.ts]: Is it used anywhere? We should not commit dead code to repository.

File: universal-login-relayer/test/helpers/WalletCreator.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  private http: HttpFunction;
+  private relayerConfig?: PublicRelayerConfig;
+  private ensService?: ENSService;
+
+  constructor(relayerUrl: string, private wallet: Wallet) {
+    this.http = http(fetch)(relayerUrl);
+  }
+
+  async getRelayerConfig(): Promise<PublicRelayerConfig> {
+    this.relayerConfig = this.relayerConfig || (await this.http('GET', '/config')).config;
```
Comment (marekkirejczyk) [inline universal-login-relayer/test/helpers/WalletCreator.ts]: Why not just inject config? We are on the server.

File: universal-login-relayer/test/helpers/WalletCreator.ts (hunk 2 above)
Comment (JustynaBroniszewska) [reply universal-login-relayer/test/helpers/WalletCreator.ts]: There is queued (on my computer) PR that I use WalletCreator in DevelopmentRelayer and TokenGrantingRelayer tests.

File: universal-login-relayer/test/helpers/WalletCreator.ts (hunk 3 above)
Comment (JustynaBroniszewska) [reply universal-login-relayer/test/helpers/WalletCreator.ts]: True, I will do that

File: universal-login-relayer/test/helpers/WalletCreator.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    const {chainSpec: {ensAddress}} = await this.relayerConfig;
+    if (this.ensService) {
+      return this.ensService;
+    }
+    this.ensService = new ENSService(ensAddress, ['mylogin.eth'], this.wallet.provider);
+    await this.ensService.start();
+    return this.ensService;
+  }
+
+  private async getInitCode (factoryAddress: string){
```
Comment (marekkirejczyk) [inline universal-login-relayer/test/helpers/WalletCreator.ts]: Unnecessary space

**Pull request merged**